network-api/networkapi/management/commands/fix_alias_page_first_last_published.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from wagtail.core.models import Page
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        aliases_without_first_last_published = (
+            Page.objects
+            .filter(alias_of__isnull=False)
+            .filter(Q(alias_of__first_published_at__isnull=False) | Q(alias_of__last_published_at__isnull=False))
+            .select_related('alias_of')
+        )
+        found = aliases_without_first_last_published.count()
+        if not found:
+            return self.stdout.write(self.style.WARNING("Found no alias pages to update"))
```

Comment:
Can we use a normal print statement here instead? This doesn't feel like a warning so much as the actual desired case.

Reply:
Done!